js/site.js

```diff
@@ -78,31 +79,33 @@ function cloudSVG(n, precip) {
   cloud_id = cloud_id+cloud_count;
   svg.setAttribute("id", cloud_id);
   cloud_count = cloud_count++;
- 
+  
   
   canvas.appendChild(svg);
 
   let position ={rad:50, xpos:500, ypos:500};
 
   for (i = 0; i < n; i++){
     position = generatePositionValues(position);
-    console.log("Cloud Color: ", cloudColor(precip));
-    document.getElementById(cloud_id).innerHTML += createCircle(position, cloudColor(precip));
+    console.log("Cloud Color: ", p);
+    document.getElementById(cloud_id).innerHTML += createCircle(position, p);
   }
-
   return svg;
 }
 
 // Returns a circle with given position and radius data
 function createCircle({rad, xpos, ypos}, p) {
   console.log("createCircle() called successfully");
+  console.log(`<circle cx="${xpos}px" cy="${ypos}px" r="${rad}px" fill="${p}"></circle>`);
   return `<circle cx="${xpos}px" cy="${ypos}px" r="${rad}px" fill="${p}"></circle>`;
 }
 
+// Sets hsl color for cloud
 function cloudColor(n) {
   let c = remapNumber(n, 0, 200, 25, 85);
   let col_val = 110 - c;
-  return `hsl(0%, 0%, ${col_val}%)`;
+  console.log(`hsl(0, 0%, ${col_val}%)`);
```

Comment:
Out of curiosity @MassterKek, are you using these console logs for debugging? If so, I'd keep them, but if you're just adding out of habit I'd recommend removing them. Just adds noise.

Reply:
They were for debugging, was going to clear them later